modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -8043,6 +8238,26 @@ type Query {
     includeDeleted: Boolean! = false
   ): AsterismGroupSelectResult!
 
+  """
+  Select a single Call for Proposals by id.
+  """
+  callForProposals(
+    callForProposalsId: CallForProposalsId!
+  ): CallForProposals
+
+  """
+  Select all Calls for Proposals.
+  """
+  callsForProposals(
+    OFFSET: CallForProposalsId
+
+    "Limits the result to at most this number of matches (but never more than 1000)."
+    LIMIT: NonNegInt
+
+    "Set to true to include deleted values"
+    includeDeleted: Boolean! = false
+  ): CallsForProposalsSelectResult!
```

Comment:
What about including an `includeClosed` option as well. For display in explore we'll only care about open CfPs.

Reply:
Yes, the plan for the next related PR is to add a `WHERE` clause with options like that and more.